ios/HathorMobile/main.m

```diff
@@ -1,10 +1,3 @@
-/**
```

Comment:
Can we remove this header?

Should we add Hathor's?

Reply:
This was removed from in the diff, so I don't see a problem with removing it but adding Hathor's would seem odd.